src/main/java/org/automation/pages/CustomersPage.java

```diff
@@ -0,0 +1,393 @@
+    By selectCustomer = By.xpath("//div[@id=\"_EH\"]");
+    By billPhonenumber = By.xpath("//input[@placeholder=\"Phone number\"]");
+    public By billPhoneGoBtn = By.xpath("//button[@class=\"btn btn-primary\"]");
+    public By billSendBtn = By.xpath("//div[@class=\"-totalDivId- d-flex justify-content-around position-relative\"]");
+    By customerDisplayed = By.xpath("//span[text()=\"Rishabh\"]");
+    By searchField = By.xpath("//input[@placeholder=\"Search\"]");
+    By searchBtn = By.xpath("(//button[@class=\"btn btn-primary\"])[3]");
+    By alertValidation = By.xpath("//h4[@class=\"alert-heading\"]");
+    By noResult = By.xpath("//div[@class='no-result-icon']");
+    public By selectCustomerinFilter = By.xpath("//div[@class='d-flex align-items-center position-relative mb-3 rounded border']");
+    By customerSelection = By.xpath("//div[@class='border rounded-3 mb-1 p-2 position-relative clone']");
+    By customerdisplayed = By.xpath("//div[@class='bg-white mb-2 row position-relative m-0 g-2 border rounded-2 -cust-row-']");
+    By addPaymentMethod = By.xpath("//button[@class='-btn-add-payment- btn-sm btn btn-link']");
+    By permissionChkbx = By.xpath("//i[@class='fal fa-square custom-check-off ']");
+    By permissionCtnBtn = By.xpath("//button[@class='checked-d-inline-block d-none -launch-add-paymentmethod- btn btn-primary']");
+    public By addCardNumberField = By.xpath("//input[@placeholder='1234 1234 1234 1234']");
+    By expirationDate = By.xpath("//input[@class='p-Input-input Input Input--empty p-Input-input--textRight']");
+    By securityCode = By.xpath("//input[@placeholder='CVC']");
+    By countrySelect = By.xpath("//select[@class=\"Input p-Select-select\"]");
+    By selectCountry = By.xpath("//option[text()=\"Australia\"]");
+    By saveCardBtn = By.xpath("//button[@class=\"btn btn-primary display-none\"]");
+    By closefilter = By.xpath("//button[@class='btn-close text-reset']");
+    By frame1 = By.xpath("//iframe[@title='Secure payment input frame']");
+
+    public void storeSelection() {
+        click(storesCombobox);
+        click(selectStore);
+    }
+
+    public Clickable continuebtn() {
+        return Clickable.getElementBy(cnt,"Continue button");
+    };
+
+    public Editable nameAddress() {
+        return Editable.getElementBy(nameAddress,"Store name and address");
+    }
+
+    public Clickable findAddCustomer() {
+        return Clickable.getElementBy(findAddCustomer,"Find or Add Customer");
+    }
+
+    public Clickable getCustPopupCloseBtn() {
+        return Clickable.getElementBy(customerPopupClose,"Close button for Customer popup");
+    }
+
+    public Clickable filter() {
+        return Clickable.getElementBy(filter,"Filter icon");
+    }
+
+    public void setPhoneNumber(String phoneNumberInput) {
+        Editable phoneField = Editable.getElementBy(phoneNumber, "Phone Number field");
```

Comment:
Good Practice: Abstracting frequently used actions like setting phone numbers or email addresses into reusable methods.
Improvement Suggestion: Combine actions for better readability, e.g., setAndSubmitPhoneNumber.

Reply:
Thank you so much for your feedback!
I did not integrate the action into a single step to ensure that the input can be modified directly from the test case page at any time.